Notebooks/README.md

```diff
@@ -7,18 +7,14 @@
 
 <table style="border-collapse: collapse;">
 	<tr>
-		<td align="center"><b>Branch</b></td>
-		<td align="center"><b>Travis</b></td>
-		<td align="center"><b>Appveyor</b></td>
+		<td align="center">CI</td>
+		<td align="center"><img src="https://github.com/ECSIM/dbfc-dataset/workflows/CI/badge.svg?branch=master"></td>
+		<td align="center"><img src="https://github.com/ECSIM/dbfc-dataset/workflows/CI/badge.svg?branch=dev"></td>
```

Comment:
There is no `dev` branch
Please remove this line

Reply:
Excuse me for this careless commit.